def/es6.js

```diff
@@ -108,13 +108,26 @@ def("SpreadElement")
     .field("argument", def("Expression"));
 
 def("ArrayExpression")
-    .field("elements", [or(def("Expression"), def("SpreadElement"), null)]);
+    .field("elements", [or(
+        def("Expression"),
+        def("SpreadElement"),
+        def("RestElement"),
+        null
+    )]);
```

Comment:
I think you'll only find a `RestElement` in an `ArrayPattern` (that is, an array expression on the left-hand side of an assignment or variable declaration).